src/content/docs/en/developers/guides/running-a-scroll-node.mdx

```diff
@@ -0,0 +1,406 @@
+---
+section: developers
+date: Last Modified
+title: "Running a Scroll L2geth Node"
+lang: "en"
+permalink: "developers/guides/running-a-scroll-node"
+whatsnext: { "Bridge ERC20 through the Custom Gateway": "/developers/guides/bridge-erc20-through-the-custom-gateway" } #TODO: update this
+excerpt: "In this example, we will launch a dummy smart contract on either Sepolia or Scroll Sepolia testnet and interact with it from the opposite chain." #TODO: update this
+---
+
+import Aside from "../../../../../components/Aside.astro"
+import ToggleElement from "../../../../../components/ToggleElement.astro"
+
+Scroll implements a fork of geth, with clique for consensus, which we call l2geth. 
+
+## Prerequisites
+
+### Finding the latest version
+
+We recommend using the latest release at https://github.com/scroll-tech/go-ethereum/releases. For Scroll Mainnet, the required version is `scroll-v5.0.0` or higher. If you'd like to keep up with new node releases, go to https://github.com/scroll-tech/go-ethereum, click on **Watch**, **Custom**, and make sure that **Releases** is selected. 
+
+For the remainder of this guide, `VERSION` will denote the version tag. For example, `scroll-v5.1.10`.
+
+### Hardware Requirements
+
+**Minimum**
+
+- Machine comparable to AWS `t3.large` [instance](https://aws.amazon.com/ec2/instance-types/t3/).
+- 500GB SSD storage.
+
+**Recommended**
+
+- Machine comparable to AWS `t3.2xlarge` [instance](https://aws.amazon.com/ec2/instance-types/t3/).
+- 1TB SSD storage.
+
+### L1 RPC Node 
+
+You will need access to a fully-synced **Ethereum Mainnet** or **Ethereum Sepolia** RPC endpoint before running `l2geth`, depending on if you would like to run a Scroll Mainnet or Scroll Sepolia node. You can maintain your own node or choose to use a 3rd-party provider that supports the standard Ethereum RPC APIs. Define your L1 node’s HTTP RPC API at `L2GETH_L1_ENDPOINT` (ie. `L2GETH_L1_ENDPOINT="https://my-node.com:8545"`).
+
+---
+
+## Build and Run From Source
+
+### Download and Build L2geth
+
+1. Clone the latest version of `l2geth`. 
+
+    ```bash
+    $ git clone https://github.com/scroll-tech/go-ethereum l2geth-source
+    $ cd l2geth-source
+    $ git checkout VERSION
+    ```    
+2. Ensure that you are using version 1.18 of `go`.
+3. Install the gcc compiler: `sudo apt install build-essential`.
+4. Build `l2geth`: `make nccc_geth`. The binary is now in `build/bin/geth`.
+5. Define a command alias: `alias l2geth=./build/bin/geth`.
+
+{/* <Aside>
+`nccc` stands for “*no circuit capacity checker*”. This feature disables an expensive block validation rule that is not crucial on follower nodes. We recommend that you do not enable it. **If you do, please let us know so that we can notify you in case of breaking ccc upgrades.**
```

Comment:
I commented this out, do we feel like it should be included?

Reply:
I don't think we need to include it.